mlf_core/create/templates/mlflow/mlflow_pytorch/{{ cookiecutter.project_slug }}/MLproject

```diff
@@ -10,25 +10,34 @@ name: {{ cookiecutter.project_slug }}
 docker_env:
   image: ghcr.io/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug_no_hyphen }}:{{ cookiecutter.version }}
   volumes: ["${PWD}/data:/data"]
+  environment: [["MLF_CORE_DOCKER_RUN", "TRUE"]]
 
 entry_points:
   main:
     parameters:
-      cuda: {type: string, default: 'True'}
-      epochs: {type: int, default: 5}
-      general-seed: {type: int, default: 0}
-      pytorch-seed: {type: int, default: 0}
+      max_epochs: {type: int, default: 5}
+      gpus: {type: int, default: 0}
+      accelerator: {type str, default: "None"}
```

Comment:
Is it possible to make this as default ddp ?
Does it then still work with the CPU and single GPU?

Reply:
I dont think this will work, so I'd leave the default as it is

Reply:
dp works with a single GPU, so ddp might as well.

Likely breaks for CPU, but would be nice to try it.